cmd/snap-bootstrap/cmd_initramfs_mounts.go

```diff
@@ -72,6 +72,10 @@ var (
 	osutilIsMounted = osutil.IsMounted
 )
 
+var (
+	diskByLabel = "/dev/disk/by-label"
```

Comment:
this should probably be boringly called:  devDiskByLabelDir

probably also a comment about it getting mocked by tests